lib/etrade/parse-etrade-gl.ts

```diff
@@ -28,9 +28,7 @@ export const parseEtradeGL = async (
         adjustedCost: row["Adjusted Cost Basis Per Share"],
         dateAcquired: toDateString(row["Date Acquired"]),
         dateSold: toDateString(row["Date Sold"]),
-        isPlanUsQualified: row["Qualified Plan"] === "Qualified",
```

Comment:
Keeping this helps being explicit when looking for US qualified plans, especially if/when there is other possible qualification. 
It does not cost much to have it, it will cost more to re-introduce it.

WDYT?

Reply:
I was under the impression that the plan could either be US qualified (and then FR non qualified) or FR qualified (and then US non qualified). Am I mistaken?

Reply:
Discussed offline and we'll switch to a `qualifiedIn: "fr" | "us"` to allow for easier branching down the line